### ISSUE TITLE
no dependencies nor content Network build failures in 'official/dnceng/dotnet-wpf-test/refs/heads/master' - '20190702.4'

### ISSUE THREAD

**Issue by robertborr** [dotnet/wpf#1158]
@dotnet-mc-bot commented on [Tue Jul 02 2019](https://github.com/dotnet/core-eng/issues/6860)

There were a set of failures during this build. Here is a summary of these:
* https://dnceng.visualstudio.com/internal/_build/results?buildId=248739
  - **Agent:** DDVSOWINAGE316
  - **Error log:** E:\A\_work\0\s\.dotnet\sdk\3.0.100-preview7-012512\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets(0,0): error NU5017: Cannot create a package that has no dependencies nor content. [E:\A\_work\0\s\packaging\Micros...

@robertborr

---

@miguep commented on [Tue Jul 02 2019](https://github.com/dotnet/core-eng/issues/6860#issuecomment-507899553)

I apologize about the noise, this is clearly something wrong with our repo, we are investigating

---

@robertborr commented on [Wed Jul 03 2019](https://github.com/dotnet/core-eng/issues/6860#issuecomment-508144388)

dup #6837  reclass sourcecode and moving to wpf

**Comment by rladuca** [comment]
I'm going to take this and figure out what is wrong in our build system.

@robertborr We're going to have a bunch of churn in our repo's engineering system in the near future.  It's possible we have a lot of breaks until things settle down.  I don't want to make noise and work for you and your team.  Is there something we can do to limit things that bubble up to you?

**Comment by robertborr** [comment]
Thanks for the warning ! 😊  You can disable telemetry till you’re actually ready.   Matt can help with that (I am not sure exactly how it is done)

**Comment by rladuca** [comment]
@robertborr Oh ok, if it's just the basic Arcade telemetry, I believe I can just knock that out of the build yml.  I'll take a look at doing that first thing so we stop generating issues, thanks!

**Comment by vatsan-madhavan** [comment]
I recommend that we keep this issue open as a way to track the work. If someone wants to report another issue in the near-future, we can reuse this issue, for e.g..

**Comment by rladuca** [comment]
Clearing estimate here as this is tracked in the children of internal task: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/944228

**Comment by rladuca** [comment]
Waiting on new VS update with fixes to DebugExe so I can investigate several failures from new test bins.
